gateway/src/main/java/discord4j/gateway/json/GatewayPayload.java

```diff
@@ -85,6 +85,10 @@ public T getData() {
         return data;
     }
 
+    public boolean isDataPresent() {
+        return data != null;
+    }
```

Comment:
This method must be marked as `@JsonIgnore` to exclude it on serialization

Reply:
Yup will do